packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -190,7 +190,7 @@ const generateImageSource = (
   height,
   toFormat,
   _fit, // We use resizingBehavior instead
-  { jpegProgressive, quality, cropFocus, backgroundColor, resizingBehavior }
+  options = {}
```

Comment:
I'm curious about this change. Can't you do
```suggestion
  { jpegProgressive, quality, cropFocus, backgroundColor, resizingBehavior } = {}
```